10/alpine/Dockerfile

```diff
@@ -40,15 +40,13 @@ RUN addgroup -g 1000 node \
     && grep " node-v$NODE_VERSION.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
     && tar -xf "node-v$NODE_VERSION.tar.xz" \
     && cd "node-v$NODE_VERSION" \
-    && ln -s /usr/bin/ccache /usr/local/bin/cc \
-    && ln -s /usr/bin/ccache /usr/local/bin/gcc \
-    && ln -s /usr/bin/ccache /usr/local/bin/g++ \
+    && export PATH="/usr/lib/ccache/bin/:$PATH" \
+    && ccache --set-config=max_size=150MB \
     && HOST_IP="$(ip route | awk '/default/ { print $3 }')" \
     && nc -v "$HOST_IP" 5678 | tar -xz -C / || true \
```

Comment:
Uhh, this is a very weird implementation that isn't going to pass review downstream, so I'd really recommend reverting that previous ccache PR and revising that following the security release next week (we're now into Thanksgiving holiday time here in the US, which means there's not really enough time now for a PR with larger changes before the security release).

Reply:
What if we make sure it's extracted under `/root/.ccache/`?